### DISCUSSION TITLE
Will fildem work on the upcoming release of Ubuntu 22.04 LTS with GNOME 42?

### DISCUSSION THREAD

**Discussion by Devansh80** [gonzaarcr/Fildem#138] [Q&A] [answered]
I've been an Ubuntu LTS user for a very long time and have been using it consistently since I got Fildem. Now I want to know whether this extension will work on Gnome 42 also or not. Because I will definitely upgrade to this upcoming release of Ubuntu LTS. And I don't want to stop using it.

**Comment by gonzaarcr** [comment, marked as answer]
I just upgraded, I’m gonna try to upgrade all my extensions.

**Reply by Devansh80** [reply]
As i can see that you have Added gnome 42 compatibility. So should i upgrade to Gnome 42 based Ubuntu 22.04 LTS?
